### DISCUSSION TITLE
miniconda  r-base=4.2.3  install openssl  get error

### DISCUSSION THREAD

**Discussion by honghh2018** [openssl/openssl#25838] [Q&A]
Hi developer,
    I want to install the openssl in R env in miniconda , unfortunately
It get error lying below:
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for ‘openssl’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/public/biosoft/miniconda3/envs/rjags-env/lib/R/library/00LOCK-openssl/00new/openssl/libs/openssl.so':
  /public/biosoft/miniconda3/envs/rjags-env/lib/R/library/00LOCK-openssl/00new/openssl/libs/openssl.so: undefined symbol: OPENSSL_init_ssl
Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/public/biosoft/miniconda3/envs/rjags-env/lib/R/library/openssl’

The downloaded source packages are in
	‘/tmp/RtmpjNzU00/downloaded_packages’
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning message:
In install.packages("openssl") :
  installation of package ‘openssl’ had non-zero exit status

The openssl version in miniconda was OpenSSL 3.3.2
R was 4.2.3

how to fix this error,
any advice would be appreciated
Best,
hanhuihon

**Comment by tom-cosgrove-arm** [comment]
This doesn't appear to be reporting an issue in OpenSSL, but asking a question about building as part of R in Miniconda - so it should really be a discussion
